components/org.wso2.carbon.identity.api.server.identity.governance/org.wso2.carbon.identity.api.server.identity.governance.v1/src/main/resources/identity-governance-3.yaml

```diff
@@ -0,0 +1,437 @@
+          $ref: '#/components/responses/ServerError'
+  '/identity-governance/{category-id}/connectors':
+    get:
+      tags:
+        - Management
+      summary: Retrieve governance connectors of a category.
+      operationId: getConnectorsOfCategory
+      description: |
+        Retrieve governance connectors of a category.
+
+        <b>Permission required:</b>
+          * /permission/admin/manage
+      parameters:
+        - $ref: '#/components/parameters/categoryIdPathParam'
+      responses:
+        '200':
+          description: Requested governance connector category.
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/ConnectorRes'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '500':
+          $ref: '#/components/responses/ServerError'
+  '/identity-governance/{category-id}/connectors/{connector-id}':
+    get:
+      tags:
+        - Management
+      summary: Retrieve governance connector.
+      operationId: getConnector
+      description: |
+        Retrieve governance connector.
+
+        <b>Permission required:</b>
+          * /permission/admin/manage
+      parameters:
+        - $ref: '#/components/parameters/categoryIdPathParam'
+        - $ref: '#/components/parameters/connectorIdPathParam'
+      responses:
+        '200':
+          description: Requested governance connector.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ConnectorRes'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '500':
+          $ref: '#/components/responses/ServerError'
+    patch:
+      tags:
+        - Management
+      summary: Patch governance connector.
+      operationId: patchConnector
+      description: |
+        Patch governance connector.
+
+        <b>Permission required:</b>
+          * /permission/admin/manage
+      parameters:
+        - $ref: '#/components/parameters/categoryIdPathParam'
+        - $ref: '#/components/parameters/connectorIdPathParam'
+      responses:
+        '200':
+          $ref: '#/components/responses/OK'
+        '400':
+          $ref: '#/components/responses/InvalidInput'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '500':
+          $ref: '#/components/responses/ServerError'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ConnectorsPatchReq'
+        description: governance-connector to update
+servers:
+  - url: >-
+      https://virtserver.swaggerhub.com/Maduranga/wso2_identity_server_governance_admin_api/1.0.0
```

Comment:
Shouldn't this be changed to default server path

Reply:
Swagger editor always change the url to this value. Thanks for pointing out. Will update the PR